src/schema/v2/artist/__tests__/helpers.test.ts

```diff
@@ -133,4 +133,58 @@ describe("getArtistInsights", () => {
       })
     })
   })
+
+  describe("gaining followers insight", () => {
+    const fields = [
+      {
+        kind: "GAINING_FOLLOWERS",
+        artist: {
+          follow_count: 200,
+          last_year_follow_count: 40,
+        },
+        returnsInsight: true,
+      },
+      {
+        kind: "GAINING_FOLLOWERS",
+        artist: {
+          follow_count: 200,
+          last_year_follow_count: 10,
+        },
+        returnsInsight: false,
+      },
+      {
+        kind: "GAINING_FOLLOWERS",
+        artist: {
+          follow_count: 200,
+          last_year_follow_count: 200,
+        },
+        returnsInsight: false,
+      },
+      {
+        kind: "GAINING_FOLLOWERS",
+        artist: {
+          follow_count: 0,
+          last_year_follow_count: 0,
+        },
+        returnsInsight: false,
+      },
+    ]
+
+    fields.forEach((field) => {
+      it(`${
+        field.returnsInsight ? "returns" : "does not return"
+      } insight when the follow_count is ${
+        field.artist.follow_count
+      } and last_year_follow_count is ${
+        field.artist.last_year_follow_count
+      }`, () => {
+        const artist = field.artist
+
+        const insights = getArtistInsights(artist)
+        const insight = insights.find((insight) => insight.kind === field.kind)!
+
+        expect(!!insight).toEqual(field.returnsInsight)
+      })
+    })
+  })
```

Comment:
**suggestion:** Add tests for exported function

Now with fewer outer loop tests and an exported function to compute the growth percent, I'd add a separate `describe` block with more focused tests for that function.

*format via [Conventional Comments](https://conventionalcomments.org/)*